engine/tester/paymailmock/mock_paymail_host_responses.go

```diff
@@ -0,0 +1,61 @@
+package paymailmock
+
+import (
+	"net/http"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/jarcoal/httpmock"
+)
+
+// MockedP2PDestinationResponse is a mocked response for the P2P destinations endpoint
+type MockedP2PDestinationResponse struct {
+	Reference string
+	Scripts   []string
+	Satoshis  []uint
+}
+
+var _mockedLockingScripts = []string{
```

Comment:
is the underscore necessary - what it indicates in this context?

Reply:
no I start with some const approach and it's leftover 🙈

Reply:
changed